### PR TITLE
Minimal SSV API

### PR DESCRIPTION
**Pull request by dankrad**
Suggested API for VC to communicated with SSV node (SSV node will emulate BN API to VC, to keep things simple)

### CODE REVIEW

**Review by hsailern — DISMISSED** [review]

File: apis/validator/ssv/stream.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      schema:
+        type: array
+        items:
+          type: string
+          uniqueItems: true
+          enum:
+            - sign_block
+            - sign_attestation
+            - check_block
+            - check_attestation
```
Comment (alonmuroch) [inline apis/validator/ssv/stream.yaml:27]: @dankrad  should include sign aggregation as well

Comment (dankrad) [reply apis/validator/ssv/stream.yaml:27]: What do you man signature aggregation? The reconstruction of the threshold signature should be done by the SSV node, the VC doesn't need to do anything about it

Comment (alonmuroch) [reply apis/validator/ssv/stream.yaml:27]: if that validator has an aggregation duty for a particular epoch we should be able to sign the proof.

Comment (dankrad) [reply apis/validator/ssv/stream.yaml:27]: Ah right, I forgot about attestation aggregation proofs. Yes, these need to be added then.

**Review by axljoker03 — APPROVED** [review]

**Comment by rolfyone** [comment]
Closing as very stale, we can revisit if it comes back around...

**Pull request closed without merging**